recipes/nextstrain-cli/meta.yaml

```
@@ -1,12 +1,12 @@
-{% set version = "8.2.0" %}
+{% set version = "8.5.0" %}
 
 package:
   name: nextstrain-cli
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/n/nextstrain-cli/nextstrain-cli-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.io/packages/source/n/nextstrain-cli/nextstrain_cli-{{ version }}.tar.gz
```
https://pypi.org/project/nextstrain-cli/#files lists https://files.pythonhosted.org/packages/9f/06/8c3c553bc94579051bbc56721c6cf22827431e07465cceec514e19654a10/nextstrain_cli-8.5.0.tar.gz